Governance/tech/conventions.md

```diff
@@ -4,76 +4,150 @@
+* Use an existing NetCDF Convention for your coordinate variables, especially to identify spatio-temporal coordinates.
+* Use shared dimensions to indicate that two variables use the same coordinates along that dimension. If two variables' dimensions are not related, create separate dimensions for them, even if they happen to have the same length. 
+```
+
+!!! other type of cocrdinates!!!
+scalar coordinate variable
+A scalar variable (i.e. one with no dimensions) that contains coordinate data. Depending on context, it may be functionally equivalent either to a size-one coordinate variable (Section 5.7, "Scalar Coordinate Variables") or to a size-one auxiliary coordinate variable (Section 6.1, "Labels" and Section 9.2, "Collections, instances, and elements").
+
+
+**Requirements**
 
-Units of year and month and any equivalent units should be used with caution.
+* The time units of a time coordinate variable must contain a reference time.
+* The reference time of a time coordinate variable must be a legal time in the specified calendar.
 
-Cliamtological statistics (check document)
+**Recommendations**
 
-```{dropdown} calendar 
-CF - recommended
-the [calendar](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.9/cf-conventions.html#calendar) attribute defines the calendar used by the time coordinate. In order to calculate a time coordinate value from a date/time, or the reverse, one must know the units attribute of the time coordinate variable (containing the time unit of the coordinate values and the reference date/time) and the calendar. The choice of calendar defines the set of dates (year-month-day combinations) which are permitted, and therefore it specifies the number of days between the times of 0:0:0 (midnight) on any two dates. Date/times which are not permitted in a given calendar are prohibited in both the encoded time coordinate values, and in the reference date/time string. It is recommended that the calendar be specified by the calendar attribute of the time coordinate variable. 
+* The use of a reference time in the year 0 to indicate climatological time is deprecated. This restriction only applies to the real-world calendar as used by the udunits package. For more information on how to describe climatological statistics refer to the [CF documentation](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.9/cf-conventions.html#climatological-statistics).
+
+* Units of year and month and any equivalent units should be used with caution.
+
+```{dropdown} **calendar** 
+**CF - recommended**<br>
+The [calendar](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.9/cf-conventions.html#calendar) attribute defines the calendar used by the time coordinate. In order to calculate a time coordinate value from a date/time, or the reverse, one must know the units attribute of the time coordinate variable (containing the time unit of the coordinate values and the reference date/time) and the calendar.<br>
```

Comment:
adding CRS in there? is sometimes used as calendar synonym

Reply:
could you create an issue and add this after I merge, I don't really know much about it :-)